tool/ood-gen/lib/is_ocaml_yet.ml

```diff
@@ -42,9 +44,9 @@ let decode (_, (head, body_md)) =
             {
               category with
               description =
-                Omd.to_html
-                  (Hilite.Md.transform
```

Comment:
Ugh, I'm just realising that Hilite works with Omd, so we'll need upstream support for Cmarkit in Hilite before we can migrate OCaml.org. There is a PR from @patricoferris for that, but I don't know if it's ready to be used: https://github.com/patricoferris/hilite/pull/11.

Reply:
oh... I see, removing Hilite here will remove syntax highlighting 🤔

Reply:
to pin hilite to that @patricoferris' upstream branch and see if it works would be a great test for that upstream PR

Reply:
@oyenuga17 in this file, we pin a dependency to a specific github commit id: https://github.com/ocaml/ocaml.org/blob/main/ocamlorg.opam.template

You can try and add a pin for `hilite` there.

Reply:
@sabine so I pinned the cmarkit commit for hilite like this
`  ["hilite.dev" "git+https://github.com/patricoferris/hilite/commit/9091776558add7dfd4fd93bfdbbab85f50a2b438"]
` in both ocamlorg.opam.template and ocamlorg.opam, but I get an error saying Hilite expects Omd.doc type
<img width="542" alt="Screenshot 2023-10-16 at 17 12 07" src="https://github.com/ocaml/ocaml.org/assets/64274826/f8e87d78-5e0a-40c9-aa81-17d66cb8bfcd">
<img width="494" alt="Screenshot 2023-10-16 at 17 13 27" src="https://github.com/ocaml/ocaml.org/assets/64274826/c29620d2-d9ab-41dc-999f-bb185f9162fd">

Reply:
you have to
1. try to build and see that `ocamlorg.opam` now contains the pinned dependency
2. run `opam install . --no-install`. If that doesn't work, create a new switch so the package gets installed.

Reply:
@sabine the first step was successful, the second wasn't. What does the --no-install flag do?, I tried it and it says `opam: unknown option '--no-install'`

Reply:
sorry, `--deps-only`

Reply:
`--no-install` is a flag that you can use when creating a switch with `opam switch create` (I mixed it up).

Reply:
@sabine I tried this but I get the following error
<img width="1305" alt="Screenshot 2023-10-17 at 14 56 21" src="https://github.com/ocaml/ocaml.org/assets/64274826/cd608a52-7ab2-4376-9d7f-ed9c7b93d2b0">
then I tried to rebuild the project to see if hilite will get installed but it didn't
<img width="622" alt="Screenshot 2023-10-17 at 14 37 40" src="https://github.com/ocaml/ocaml.org/assets/64274826/b07a7877-dc03-4ccd-aece-f87f9f42ecdb">

Reply:
I haven't tried, but it might have to be like this:

```
git+https://github.com/patricoferris/hilite#COMMIT_ID
```

Reply:
@sabine I just tried this now and it got pinned 😅 but when I try to access it in the project it still says Hilite is an unbound module like above. 
Sorry for the long conversion on this issue 😅 🙏

Reply:
@sabine so i added hilite to `dune-project` without specifying the version just like it was done for `tailwind` and `olinkcheck` then the error message changed to this
<img width="621" alt="Screenshot 2023-10-18 at 10 50 54" src="https://github.com/ocaml/ocaml.org/assets/64274826/b6834fad-d748-4051-84ab-9c8ce4903019">
I noticed when I go to
https://github.com/patricoferris/hilite#9091776558add7dfd4fd93bfdbbab85f50a2b438 using the hash style, it lands on the github home page of hilite. Could it be that the pinned hilite still references the `main` branch of hilite (omd version)?

This is the link to the actual commit for cmarkit in hilite https://github.com/patricoferris/hilite/commit/9091776558add7dfd4fd93bfdbbab85f50a2b438

Reply:
I finally was able to figure this out 😅 
The pin existed but it wasn't actually installed. Had to run `opam upgrade hilite hilite`
Followed this [post](https://discuss.ocaml.org/t/how-to-pin-a-development-repo-in-an-opam-file/12795/5).

Reply:
That is interesting. I think it will be helpful to push your current state to this branch if you want me to have a look.

Reply:
@sabine I just pushed my current state, please have a look 😊

Reply:
Hi @sabine, just want to know if you've had time to look at this 😊

Reply:
You're probably right in that the upstream branch isn't really ready yet. I think there's two things we can do:

(1) go upstream and see if contributing is possible
(2) leave PR in this state until upstream release happens

Reply:
Alright. Will leave this PR for now and probably try to see if I'm able to figure out a hack for that upstream branch. If successful, I'll create a PR and temporarily pin my version till the upstream release happens.

Reply:
It's probably a good idea to open an issue upstream to ask if Patrick has any pointers. :slightly_smiling_face: